packages/ui/src/components/theme/styleOverrides/utils/getContainerHeight.ts

```diff
@@ -0,0 +1,27 @@
+export const TOOLBAR_HEIGHT = 4;
+export const TOOLBAR_HEIGHT_SM = 3.5;
+export const PAGE_HEADER_BREADCRUMBS_HEIGHT = 6.87;
+export const PAGE_HEADER_HEIGHT = 4.87;
+export const TABS_CONTAINER_HEIGHT = 3;
+
+export const getContainerHeight = (hasToolbar: boolean, hasPageHeaderBreadcrumbs: boolean, hasTabs: boolean) => {
```

Comment:
Those arguments can be a union type, e.g. `const ContainerVariant = 'tabbed' | 'breadcrumbs' | 'toolbar'` and use that in the function to receive only one arg, the variant of the container

Reply:
okay, then let's convert them into an object so its easier to read